scrapy/pipelines/images.py

```diff
@@ -46,6 +46,7 @@ class ImagesPipeline(FilesPipeline):
     THUMBS = {}
     DEFAULT_IMAGES_URLS_FIELD = 'image_urls'
     DEFAULT_IMAGES_RESULT_FIELD = 'images'
+    IMAGES_S3_SECURE_CONNECTION = False
```

Comment:
I would prefer that no new upper-case class attribute is introduced.
It does not seem needed if `IMAGES_S3_SECURE_CONNECTION` setting is now defined.